apps/frontend/src/dashboard/__tests__/Dashboard.test.tsx

```diff
@@ -45,53 +51,39 @@ const renderDashboard = async (user: Partial<User>) => {
 };
 
 it('renders dashboard header', async () => {
-  nock(API_BASE_URL)
-    .get('/dashboard')
-    .reply(200, {
-      newsAndEvents: [],
-      pages: [],
-    } as DashboardResponse);
-
   const { findByText } = await renderDashboard({});
   expect(await findByText(/welcome/i, { selector: 'h1' })).toBeVisible();
 });
 
 it('renders dashboard with news and events', async () => {
-  nock(API_BASE_URL)
-    .get('/dashboard')
-    .reply(200, {
-      newsAndEvents: [
-        {
-          id: '55724942-3408-4ad6-9a73-14b92226ffb6',
-          created: '2020-09-07T17:36:54Z',
-          title: 'News Title',
-          type: 'News',
-        },
-        {
-          id: '55724942-3408-4ad6-9a73-14b92226ffb77',
-          created: '2020-09-07T17:36:54Z',
-          title: 'Event Title',
-          type: 'Event',
-        },
-      ],
-      pages: [],
-    } as DashboardResponse);
+  mockGetDashboard.mockResolvedValue({
+    newsAndEvents: [
+      {
+        id: '55724942-3408-4ad6-9a73-14b92226ffb6',
+        created: '2020-09-07T17:36:54Z',
+        title: 'News Title',
+        type: 'News',
+      },
+      {
+        id: '55724942-3408-4ad6-9a73-14b92226ffb77',
+        created: '2020-09-07T17:36:54Z',
+        title: 'Event Title',
+        type: 'Event',
+      },
+    ],
+    pages: [],
+  });
 
-  const { queryAllByText, findByText } = await renderDashboard({
+  const { findByText, queryAllByText } = await renderDashboard({
     firstName: 'John',
   });
+
   expect(await findByText(/john/i, { selector: 'h1' })).toBeVisible();
   expect(queryAllByText(/title/i, { selector: 'h2' }).length).toBe(2);
 });
 
 it('prefetches research outputs (REGRESSION)', async () => {
-  nock(API_BASE_URL)
-    .get('/dashboard')
-    .reply(200, {
-      newsAndEvents: [],
-      pages: [],
-    } as DashboardResponse)
-    .persist();
+  mockGetDashboard.mockResolvedValue({ newsAndEvents: [], pages: [] });
```

Comment:
```suggestion
```
You should be able to omit this line since there is a default mock in that __mocks__ directory and its not the focus of this test.